lib/MusicBrainz/Server/Entity/URL/Tsutaya.pm

```diff
@@ -0,0 +1,21 @@
+package MusicBrainz::Server::Entity::URL::Tower;
```

Comment:
You forgot to change the package name :)

Reply:
D'oh! 🤦‍♂️ I was probably looking at the wrong window while I was typing.

Reply:
This is done.